testing/trino-product-tests-launcher/bin/run-launcher

```diff
@@ -1,17 +1,18 @@
 #!/usr/bin/env bash
 
 target="${BASH_SOURCE%/*/*}/target"
-launcher_jar=( "${target}"/trino-product-tests-launcher-*-executable.jar )
 
-if test "${#launcher_jar[@]}" -ne 1; then
-    echo "Found ${#launcher_jar[@]} launcher jars in ${target}: ${launcher_jar[*]}" >&2
-    echo "Run \`./mvnw clean install -pl :trino-product-tests-launcher\`" >&2
-    exit 3
+if which mvnd >/dev/null; then
```

Comment:
nit: 

```suggestion
if ! command -v mvnd >/dev/null; then
```

Reply:
just curious, why? is it because `command` is builtin?

Reply:
yeah, that's it. It is a nitpick after all. 😄